### DISCUSSION TITLE
Implementation of filter by language region.

### DISCUSSION THREAD

**Discussion by mssoliveira** [dailydotdev/daily#806] [Feature Requests]
I would like to have a filter where I filter the publication through my region or language.

Thanks for the project.

**Comment by idoshamun** [comment]
Hi @mssoliveira,

We officially support only English content at this stage of the product.
We hope to expand to more languages in the future.

**Comment by Skulard** [comment]
i am for a change or adjustment for language filters.
as more and more people using this app will also add more different language to the automated system or suggestions.
dev.to is already seeing more japanese content which is also landing on daily.dev

**Reply by idoshamun** [reply]
we'll soon add english only content filter to make sure only english content is surfaced. we may consider adding other languages as opt-in but it's out of our scope for now
